Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
@bbriggs sorry for my confusing comment,  what I meant is that we need the call to `scripts/install_prerequisites.sh`

in this script we basically have all the dependencies 

here's what it looks like:

```bash
sudo apt-get update
sudo apt-get install curl
sudo apt-get install git
sudo apt-get install openjdk-7-jre
sudo apt-get install python-setuptools
sudo apt-get install python-dev
sudo apt-get install python-pip
sudo apt-get install unzip

sudo pip install --upgrade pip
```

So if you could execute that script in the setup prior to execute `start.sh` it should work